### ISSUE TITLE
Styling Issue of Dropdown - Options menu

### ISSUE THREAD

**Issue by IamViditAgarwal** [jupyter-widgets/ipywidgets#2255]
I have created a Dropdown widget using Ipywidgets library of Python , The Dropdown is rendered as the Select HTML tag by the browser on screen, Now I want to customize its style according to my CSS. I have styled the Select tag but facing issue in styling the <option> tag especially. I want to change the background color of <option> tag on **: hover**, which I am not able to do. 

For this Issue: I have searched, and got the feedback as `You cannot style options tag bcz it is rendered by the OS, not HTML`. So is there a way by _Ipywidgets lib or a custom setting_ that by which I can custom style <option> tag or even **create my own <select>** tag.

**Comment by maartenbreddels** [comment]
Absolutely, there are many custom libraries out there, ipyvolume, bqplot, pythreejs, ipyleaflet, and many more.
You can start by reading https://ipywidgets.readthedocs.io/en/latest/examples/Widget%20Custom.html
